dev/conformance/runner.ts

```diff
@@ -514,17 +521,91 @@ function runTest(spec: ConformanceProto) {
   );
 }
 
+/**
+ * Parses through the TestFile and changes all instances of Timestamps encoded
+ * as strings to a proper protobuf type since protobufJS does not support it at
+ * the moment.
+ */
+function updateTimeStamp(obj: {[key: string]: {}}) {
```

Comment:
Nit: `updateTimestamp` (in my opinion)

You could also use `normalizeTimestamp`, which is a concept we just introduced to the JS SDK.